harness/temporalHelpers.js

```diff
@@ -56,6 +56,20 @@ var TemporalHelpers = {
     assert.sameValue(duration.nanoseconds, nanoseconds, `${description} nanoseconds result`);
   },
 
+  /*
+   * assertDateDuration(duration, years, months, weeks, days, [, description]):
+   *
+   * Shorthand for asserting that each date field of a Temporal.Duration is
+   * equal to an expected value.
+   */
+  assertDateDuration(duration, years, months, weeks, days, description = "") {
+    assert(duration instanceof Temporal.Duration, `${description} instanceof`);
+    assert.sameValue(duration.years, years, `${description} years result`);
+    assert.sameValue(duration.months, months, `${description} months result`);
+    assert.sameValue(duration.weeks, weeks, `${description} weeks result`);
+    assert.sameValue(duration.days, days, `${description} days result`);
+  },
```

Comment:
I would expect this to also test that the other fields are zero.

Reply:
Fixed! Also rebased to get rid of the failing Graals action.